src/host_interface/virtio_netdev.c

```diff
@@ -29,6 +29,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/mman.h>
+#include <time.h>
```

Comment:
```suggestion
```

This is not needed if you're not doing `srand(time(NULL))` (which you shouldn't be doing).